geoportal/c2cgeoportal_geoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl

```diff
@@ -13,6 +13,9 @@ Information
    a `.env.mako` file where we have the environment variables used in the Docker compose files,
    `COMPOSE_PROJECT_NAME` contains the default name of the project.
 
+2. Now you have a page with all the services usefull for an administrator here:
```

Comment:
A page containing services useful for an administrator is now available here:

Reply:
Thanks :-)